docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,207 @@ The ACS will be used for V&V testing. It is defined as:
+#. Many workers might have multiple jobs simultaneously and receive multiple forms. This is not included in the current model. 
+#. Elderly people can still have to file taxes based on social security payments, but would likely not have an employer in our model. 
+
+
+1040 Form
+'''''''''
+
+**When to Sample** 
+
+- Sample compiled on the time step containing April 15th of each year 
+- Sample on a single time step for now 
+
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+    - Notes 
+  * - First name
+    - 
+  * - Middle initial 
+    - 
+  * - Last name
+    - 
+  * - Age
+    -  
+  * - DOB
+    -  
+  * - Mailing Address
+    -  
+  * - Social Security Number 
+    -
+  * - Income 
+    -
+  * - Employer ID
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Address 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Zip Code 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Joint Filer or Dependent 
+    - Identify the below simulant as a joint filer or a dependent. Can have multiple rows if multiple simulants are included on the tax documents (e.g., for a simulant with a spouse and 2 children, there will be 1 joint filer and 2 dependents) 
+  * - First name 
+    - 
+  * - Middle initial 
+    - 
+  * - Last name 
+    - 
+  * - Age in years  
+    - 
+
+
+
+**Who to Sample** 
+
+Not everyone who receives a W2 or 1099 will end up filing taxes. 
+However, those who do not are concentrated in low incomes for whom 
+taxes are not required. Currently, we will chose to have all those 
+who are legally required to file taxes, file taxes. This is a 
+limitation and is listed below. 
+
+For simulants that receive below the minimum income, 42.14% will 
+still file taxes. [Cilke_1998]_ The remainder will not. The minimum 
+income is based on the household structure and if listed in the table below. 
+
+.. list-table:: Minimum Income  
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Simulant Type 
+    - Minimum Income 
+  * - Single filing, under 65 
+    - $12,550 
+  * - Single filing, over 65 
+    - $14,250
+  * - Married joint filing, both under 65 
+    - $25,100
+  * - Married joint filing, one under 65 
+    - $26,450
+  * - Married joint filing, both over 65 
+    - $27,800
+  * - Married separate filing 
+    - $5
+
+
+Based on the household structure, the following rules can be applied 
+for who files taxes: 
+
+- Assume that 95% of spouses file jointly, this can be randomly assigned. [Nolo]_ Others will file separately. 
```

Comment:
we have the following "relationship to household head" labels for simulants:
    20: "Reference person",
    21: "Opp-sex spouse",
    22: "Opp-sex partner",
    23: "Same-sex spouse",
    24: "Same-sex partner",
    25: "Biological child",
    26: "Adopted child",
    27: "Stepchild",
    28: "Sibling",
    29: "Parent",
    30: "Grandchild",
    31: "Parent-in-law",
    32: "Child-in-law",
    33: "Other relative",
    34: "Roommate",
    35: "Foster child",
    36: "Other nonrelative",
    37: "Institutionalized GQ pop",
    38: "Noninstitutionalized GQ pop",

I assume but want to confirm --- the set of all "spouses" in the simulation on any given timestep is the set of all pairs of [Reference person, Opp-sex spouse], [Reference person, Same-sex spouse] in households, ignoring if there are, for example: two parents in a household, two "other nonrelatives" (who conceivably could be married) in a household, etc. ?

^ and from that set of people we choose 95% of them to file jointly?

Reply:
also: do you want there to be year-to-year persistence in who files jointly?

Reply:
^ if so, do you want us to do anything to ensure it's always about 95% of spouses year to year that are filing jointly?

Reply:
Yes, the only spouses we will cover are [Reference person, Opp-sex spouse] and [Reference person, Same-sex spouse]. I agree there could be other married couples within the house, our current system doesn't work particularly well for complex family structures. I will clarify in the docs. 

I think lets do random for filing jointly as I imagine that is simpler. Again, I will add to the docs.